vulnerabilities/importers/redhat.py

```diff
@@ -75,21 +78,50 @@ def to_advisory(advisory_data):
     references = []
     if advisory_data.get("bugzilla"):
         bugzilla = advisory_data.get("bugzilla")
+        url = "https://bugzilla.redhat.com/show_bug.cgi?id={}".format(bugzilla)
+        bugzilla_severity = VulnerabilitySeverity(
+            severity_type="REDHAT_BUGZILLA_SEVERITY",
```

Comment:
My best effort is RH_BUGZILLA_SEVERITY which I guess is not enough, please do the naming

Reply:
What are you trying to convey there? I think that this is information about how the severity score value was "formed" or "computed". `CVSSv2` and `CVVSv3` are good examples, but then they are just codes.
So either:
1. we have codes that could eventually be related to more details likely in a table with some texts, urls, etc. 
2. OR this type is self-standing and should be as explicit as possible in a short string. REDHAT_BUGZILLA_SEVERITY and RH_BUGZILLA_SEVERITY does not tell me much. I would need to do quite a bit searching to find what you meant.
The best I can think of if we have a single string for the type would be to use a URL such as 
- https://bugzilla.redhat.com/page.cgi?id=fields.html#bug_severity
- https://www.first.org/cvss/v3-0/
- https://www.first.org/cvss/v2/

IMHO the 1st solution may be better but that's weak

Reply:
What's the difference in (1) and (2) if you are using urls either directly or indirectly (via code-> mapping) ? 

How would (1) solve the problem which you had with (2) ? 
>REDHAT_BUGZILLA_SEVERITY and RH_BUGZILLA_SEVERITY does not tell me much. I would need to do quite a bit searching to find what you meant.

It would only make it worse, since now there's 100% chance you need to lookup what the code means. 

I'm more inclined towards either using something like `REDHAT_BUGZILLA_SEVERITY` or the urls you mentioned.

Reply:
This resolved, we chose to have a in-memory registry of scoring systems